client/src/components/SplitList.jsx

```diff
@@ -1,100 +1,137 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import IndividualOwes from './IndividualOwes.jsx';
+import axios from 'axios';
+// import Dotenv from 'dotenv-webpack';
 
 const SplitList = (props) => {
+// change this to hold its own state for tip?
+
   const numberOfGuests = props.guests.length;
   const billTotalWithoutTipOrTax = props.totalCost;
   const tipPercentage = props.tipPercentage;
   const tipAmount = billTotalWithoutTipOrTax * tipPercentage / 100;
   const billWithTip = billTotalWithoutTipOrTax + tipAmount;
   // API CALL FOR TIP PERCENTAGE
-  const taxPercentage = 7;
-
-  const taxAmount = billTotalWithoutTipOrTax * taxPercentage / 100;
-  const billWithTipAndTax = billWithTip + taxAmount;
-
-  let paymentsOwed;
+  let taxPercentage;
+  const [taxDecimal, setTaxDecimal] = useState(0);
 
-  const getGuestTotals = (guests) => {
-    const totals = {};
-    const guestArray = guests;
-    const guestIDs = [];
-    for (let i = 0; i < guestArray.length; i++) {
-      const currentGuestID = guestArray[i].guestName;
-      // const currentGuestID = guestArray[i].guest_id;
-      guestIDs.push(currentGuestID);
-      totals[currentGuestID] = 0;
-      const currentGuestOrders = guestArray[i].order;
-      for (let j = 0; j < currentGuestOrders.length; j++) {
-        const orderItemCost = currentGuestOrders[j].price;
-        const howManyOrdered = 1;
-        // const howManyOrdered = currentGuestOrders[j].qty;
-        const itemTotal = orderItemCost * howManyOrdered;
-        totals[currentGuestID] += itemTotal;
+  useEffect(() => {
+    console.log('💸 💸 💸 💸 💸 SPENT A PENNY 💸 💸 💸 💸 💸 💸 💸 ');
+    const options = {
+      method: 'GET',
+      url: `https://u-s-a-sales-taxes-per-zip-code.p.rapidapi.com/${props.zipCode}`,
```

Comment:
Do you want to just save some results as example data so you don't get charged per API call? Or just setup a random number generator for now? We could always add the API Call back in before the final presentation if we want the real tax data.

Reply:
Changed! Good call thank you!